src/styles/mixin/card.scss

```diff
@@ -0,0 +1,117 @@
+$page-font-family: Roboto, sans-serif;
+$font-weigth-bolt: 700;
+$font-weigth-normal: 400;
+$primary-color: #060b35;
+$secondary-color: #616070;
+$main-width: 166px;
+$text-color: #fff;
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-family: $page-font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-color;
+    align-content: space-evenly;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-family: $page-font-family;
+    font-size: 10px;
+    font-weight: $font-weigth-normal;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    align-content: flex-start;
+  }
+
+  &__active {
+    display: flex;
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: $page-font-family;
+    font-weight: $font-weigth-normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: $primary-color;
+    margin: 0;
+    margin-top: 2px;
+  }
+
+  &__value {
+    justify-content: space-between;
+    display: flex;
+    height: 18px;
+  }
+
+  &__price {
+    font-family: $page-font-family;
+    font-size: 12px;
+    font-weight: $font-weigth-normal;
+    line-height: 18px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__price-number {
+    font-family: $page-font-family;
+    font-weight: $font-weigth-bolt;
+    font-size: 16px;
+    line-height: 18px;
+    color: #000;
+    align-items: end;
```

Comment:
The property `align-items` is not applicable to the `.card__price-number` element as it is not a flex container. Consider using `vertical-align` or another appropriate property if alignment is needed.